Firmware/RTK_Surveyor/Display.ino

```diff
@@ -444,7 +444,7 @@ void updateDisplay()
       }
 
       //Top right corner
-      if (icons & ICON_BATTERY)
+      if ((icons & ICON_BATTERY) && (productVariant != REFERENCE_STATION))
```

Comment:
So what is your plan for graphics in that corner? IP address? Probably not enough space. But we will need a displayIPAddress screen someday very soon.

Reply:
I was thinking about an Ethernet-link icon?

Reply:
Excellent idea. TBD (to be drawn).